### PR TITLE
Filling empty test files

### PR DESCRIPTION
**Pull request by DmitryGorokhov**
## Proposed changes

This opportunity appears after creating test files for missing ao classes in pr #5731.
Added unit tests for the following classes:
- `ListViewItemListAccessibleObject`
- `ListViewItemTileAccessibleObject`
- `DomainItemListAccessibleObject`
- `DomainItemAccessibleObject`
- `ScrollBarChildAccessibleObject`
- `TrackBarChildAccessibleObject`

## Customer Impact

- No

## Regression? 

- No

## Risk

- Minimal

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/5763)

### CODE REVIEW

**Comment by DmitryGorokhov** [comment]
There are changes in 3 commits: just for your easier review :) I'll combine it in 1 commit before merge

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DomainUpDown.DomainItemListAccessibleObjectTests.cs [hunk 1]
```diff
@@ -16,5 +16,58 @@ public void DomainItemListAccessibleObject_Ctor_Default()
             Assert.Null(accessibleObject.Parent);
             Assert.Equal(AccessibleRole.List, accessibleObject.Role);
         }
+
+        [WinFormsFact]
+        public void DomainItemListAccessibleObject_Name_Default_ReturnsExpected()
+        {
+            var accessibleObject = new DomainItemListAccessibleObject(null);
+            Assert.Equal("Items", accessibleObject.Name);
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DomainUpDown.DomainItemListAccessibleObjectTests.cs:24]: nit: just to make this test robust, its better to name this string constant in `DomainItemListAccessibleObject `class and reuse it here.

**Review by RussKie — APPROVED** [review]

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DomainUpDown.DomainItemListAccessibleObjectTests.cs (hunk 1 above)
Comment (DmitryGorokhov) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DomainUpDown.DomainItemListAccessibleObjectTests.cs]: Thanks, added

**Comment by DmitryGorokhov** [comment]
Combined into 1 commit and prepared to merge

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**